library/options.json

```diff
@@ -0,0 +1,308 @@
+{
+  "scene": {
+    "up_direction": {
+      "type": "string",
+      "default_value": "+Y"
+    },
+    "animation": {
+      "autoplay": {
+        "type": "bool",
+        "default_value": "false"
+      },
+      "index": {
+        "type": "int",
+        "default_value": "0"
+      },
+      "speed_factor": {
+        "type": "ratio",
```

Comment:
we never said that. a ratio will support being parsed as percentage, eg: "150%". That all it will do afaics. I dont see value in restricting ratio to 0-1 fpr the users.

Reply:
Fair enough